tests/pom.xml

```diff
@@ -58,6 +58,9 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
     <!-- logging profile: either prod, dev or trace -->
     <logging.profile>prod</logging.profile>
     <trace.frames>0</trace.frames>
+    <jaeger.disabled>true</jaeger.disabled>
```

Comment:
can we rename this to *jaeger.enabled*? I find it much easier to understand boolean properties that are expressed in their original positive semantics ...

Reply:
That was my initial approach. However that would require quite some gymnastics to invert the value for the `<skip>` field in the Jaeger docker image run configuration. I think that is not worth the effort, especially since the `jaeger.disabled` field is not supposed to be set on the command line. Instead the `jaeger` maven profile should be used.

Reply:
I see, makes sense